weather.py

```diff
@@ -57,7 +64,25 @@ def load_data_from_csv(csv_file):
     Returns:
         A list of lists, where each sublist is a (non-empty) line in the csv file.
     """
-    pass
+    with open(csv_file) as csv_data:
+        reader = csv.reader(csv_data)   
+        data_from_csv = list(reader) #how to make it skip the first line
+        data_from_csv.pop(0) #remove element in pos 0
+
+        my_result = []
+        for line in data_from_csv:
+            print(line)
+            if line == []:
+                pass
+            else:
```

Comment:
Nice approach, this works well for smaller files. Another approach is to use `next(reader)` to skip the header. This only really helps when you work with massive files, to avoid needing to perform `list(reader)` (can be expensive), followed by another `for` loop through all the data.